src/sentry/notifications/notifications/rules.py

```diff
@@ -173,7 +181,11 @@ def __init__(
         last_release: Optional[Release] = None,
     ) -> None:
         super().__init__(notification, target_type, target_identifier)
-        self.last_release = last_release
+        self.last_release = (
+            last_release
+            if last_release
+            else ActiveReleaseEventCondition.latest_release(notification.event)
+        )
```

Comment:
I believe the circular import was happening in this line:
https://github.com/getsentry/getsentry/runs/7124256782?check_suite_focus=true#step:5:778

I'm surprised this passed in `sentry` CI, but completely fell flat on the `getsentry` CI.